script/release

```diff
@@ -82,9 +75,6 @@ verbose("npm version #{version_string} --no-git-tag-version")
 
 verbose('git add -A')
 verbose("git commit -m 'Publish #{tag}'")
-verbose("git tag -d #{tag} || true")
-verbose("git tag #{tag}")
-verbose("git tag #{semvertag} -m \"Release #{semvertag}.\"")
```

Comment:
Where do we tag our releases after this change?

Reply:
How can we detect when we are attempting to re-release the same version number? (`npm publish` will fail, but can we detect it earlier?)